.github/workflows/test.yml

```
@@ -0,0 +1,24 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  test:
```

The job must have a name as per our convention
> the name attribute for job must be provided as this is the value which will be used in the web UI
```suggestion
  test:
    name: Run Tests
```

---

is this resolved?

---

yeah, i missed that https://github.com/nimblehq/infrastructure-templates/pull/22/commits/a89d823981614c070c2354964efc9293963a116d